recipes-kernel/linux/files/defconfig

```diff
@@ -0,0 +1,882 @@
+CONFIG_SYSVIPC=y
```

Comment:
Can you briefly explain how this file get generated and what changes have you made on top of the toradex kernel config?

Also, should we name it as smart_racks_defconfig?

Reply:
So far we haven't made any changes to the default kernel config yet (apart from an extra comment at the end of the file, to identify that this defconfig was used instead of the default one). We can change it to a different name, but this would most likely require some additional changes to be specified in our bbappend recipe as well. The KBUILD_DEFCONFIG_mx8 attribute in [Toradex's kernel recipe](https://git.toradex.com/cgit/meta-toradex-nxp.git/tree/recipes-kernel/linux/linux-toradex_5.4-2.1.x.bb?h=dunfell-5.x.y) points to 'defconfig' by default, so we'll have to try to change that.